backend/corpora/dataset_processing/h5ad_data_file.py

```diff
@@ -289,34 +287,22 @@ def get_corpora_properties(self):
 
         return corpora_props
 
-    def get_corpora_versions(self):
+    def get_corpora_schema_version(self):
         """
         Given an AnnData object, return:
             * None - if not a Corpora object
-            * [ corpora_schema_version, corpora_encoding_version ] - if a Corpora object
+            * schema_version (str) - if a Corpora object
 
         Implements the identification protocol defined in the specification.
         """
 
         # Per Corpora AnnData spec, this is a Corpora file if the following is true
-        if "version" not in self.anndata.uns_keys():
+        if "schema_version" not in self.anndata.uns_keys():
             return None
 
-        version = self.anndata.uns["version"]
-        if not isinstance(version, collections.abc.Mapping) or "corpora_schema_version" not in version:
-            return None
-
-        corpora_schema_version = version.get("corpora_schema_version")
-        corpora_encoding_version = version.get("corpora_encoding_version")
+        schema_version = self.anndata.uns["schema_version"]
+        if validate_version_str(schema_version):
+            return schema_version
 
-        # Validate that each of the version numbers adhere to SEMVAR format.
-        if validate_version_str(corpora_schema_version) and validate_version_str(corpora_encoding_version):
-            return [corpora_schema_version, corpora_encoding_version]
-
-    def corpora_is_version_supported(self, corpora_schema_version, corpora_encoding_version):
-        return (
-            corpora_schema_version
-            and corpora_encoding_version
-            and corpora_schema_version.startswith("1.")
-            and corpora_encoding_version.startswith("0.1.")
-        )
+    def corpora_is_schema_version_supported(self, schema_version):
+        return schema_version and (schema_version.startswith("1.") or schema_version.startswith("2."))
```

Comment:
I think we can safely assume that every dataset we ingest following the merge of the epic will be schema 2.0.0

Reply:
If not, we should write a ticket and put a TODO here for when we no longer allow 1.x.x datasets